src/Compilers/CSharp/Portable/Parser/DocumentationCommentParser.cs

```diff
@@ -288,6 +288,17 @@ private XmlNodeSyntax ParseXmlElement()
                 _pool.Free(attrs);
             }
         }
+
+        private static bool MatchingXmlNames(XmlNameSyntax name, XmlNameSyntax endName)
+        {
+            // NOTE we do not compare colon tokens.
+            //      as long as name and prefix are matching we will consider names equivalent
+            //      it would be a different error if colon are missing
+            return name.FullWidth == endName.FullWidth &&
```

Comment:
no, in error cases it will also include whitespaces.
In a valid case fullwidth will have to match.

Whitespaces are not allowed here, so the fullwidth can differ (when name and prefix match) only when the xml names are malformed. 
I figured from the comments on the previous iteration that there is a desire to report mismatched begin/end errors even if tags differ by extra whitespaces. This is mostly about reporting more or less errors.

I could also compare the content of whitespaces, which are by themselves an error, but figured that it would not be helpful to anybody.